src/Helpers/index.js

```diff
@@ -171,7 +174,7 @@ export function drawGeoJSON(map, id, data, paint = {}, onClick) {
 
         map.addLayer({
             id: layerId,
-            type: 'fill',
+            type,
```

Comment:
Will mapbox throw some kind of error if the type is un-recognized ? Should we try catch or validate the type ? I didn't find anything in the mapbox docs regarding this.

Reply:
It's define [here](https://docs.mapbox.com/mapbox-gl-js/style-spec/#layers) but I think it's better to keep it as a wrapper and not starting to duplicate checks made in the underlying library.